src/app/pages/database-details/database-details.html

```
@@ -0,0 +1,194 @@
+              </td>
+            </tr>
+            } @if (!availabilityRows.length) {
+            <tr>
+              <td colspan="5" class="px-5 py-6 text-center text-gray-400">
+                No prices available for this database.
+              </td>
+            </tr>
+            }
+          </tbody>
+        </table>
+        @if (availabilityRows.length > 7 && !expandedCards.availability) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)